Source/JavaScriptCore/wasm/WasmSlowPaths.cpp

```diff
@@ -848,6 +853,16 @@ WASM_SLOW_PATH_DECL(set_global_ref)
     WASM_END_IMPL();
 }
 
+extern "C" uint64_t ipintGetGlobal64B(Wasm::Instance* instance, unsigned index)
```

Comment:
Let's use some kind of macro for these kinds of operations (similar to how llint operations are declared)